apps/financial-aid/web-veita/src/components/Profile/MunicipalityProfile.tsx

```diff
@@ -0,0 +1,256 @@
+          TextTableItem('small', municipality.individualAid.livesWithParents),
+          TextTableItem('small', municipality.cohabitationAid.livesWithParents),
+        ]
+      case AidType.UNKNOWN:
+        return [
+          TextTableItem('h5', 'Ekkert að ofantöldu'),
+          TextTableItem('small', municipality.individualAid.unknown),
+          TextTableItem('small', municipality.cohabitationAid.unknown),
+        ]
+      default:
+        return [<></>]
+    }
+  }
+  return (
+    <Box
+      marginTop={15}
+      marginBottom={15}
+      className={`${styles.applicantWrapper} ${styles.widthAlmostFull}`}
+    >
+      <Box className={`${styles.widthAlmostFull}`}>
+        <Box className={`contentUp delay-25`} marginBottom={[3, 3, 7]}>
+          <Text as="h1" variant="h1" marginBottom={2}>
+            {municipality.name}
+          </Text>
+
+          <Box display="flex" marginRight={1} marginTop={5}>
+            <Box marginRight={1}>
+              <Text variant="small" fontWeight="semiBold" color="dark300">
+                Staða
+              </Text>
+            </Box>
+            <Box marginRight={1}>
+              <Text variant="small">
+                Sveitarfélag er {municipality.active ? 'virkt' : 'óvirkt'}
+              </Text>
+            </Box>
+            <button
+              onClick={() => console.log('bla')}
+              className={headerStyles.button}
+            >
+              {municipality.active ? 'Óvirkja' : 'Virkja'}
+            </button>
+          </Box>
+        </Box>
+        <Box marginBottom={7}>
+          <Box marginBottom={3}>
+            <Text as="h3" variant="h3" color="dark300">
+              Stjórnendur
+            </Text>
+          </Box>
+
+          <div className={`${tableStyles.smallTableWrapper} hideScrollBar`}>
+            <table
+              className={cn({
+                [`${tableStyles.tableContainer}`]: true,
+              })}
+            >
+              <thead className={`contentUp delay-50`}>
+                <tr>
+                  {['Nafn', 'Kennitala', 'Netfang', 'Aðgerð'].map(
+                    (item, index) => (
+                      <TableHeaders
+                        header={{ title: item }}
+                        index={index}
+                        key={`usersTableHeaders-${index}`}
+                      />
+                    ),
+                  )}
+                </tr>
+              </thead>
+
+              <tbody>
+                {municipality.adminUsers?.map((item: Staff, index) => (
+                  <TableBody
+                    items={[
+                      TextTableItem('h5', item.name),
+                      TextTableItem('small', item.nationalId),
+                      TextTableItem('small', item.email),
+                      ActivationButtonTableItem(
+                        'Óvirkja',
+                        false,
+                        () => console.log('bla'),
```

Comment:
is this some theme, instead of bla, maybe todo? 🤓

Reply:
You don't like the `bla`????? 😂